changehc/delphi_changehc/download_ftp_files.py

```diff
@@ -93,3 +93,30 @@ def download_cli(filedate, out_path, ftp_conn):
     finally:
         if client:
             client.close()
+
+
+def download_flu(filedate, out_path, ftp_conn):
```

Comment:
What's the difference between download_flu, download_covid, and download_cli? If they're identical I'd rather we not copy-paste.